src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -509,6 +502,61 @@ export default Vue.extend({
       }
     },
 
+    toggleFullWindowed: function() {
```

Comment:
Let's avoid creating a function within an existing function.  This should be possible to branch out into a `clickFullWindow` function within the component.  Alternatively, the new function can be called `toggleFullWindow` because of my next comment.

This function seems like it isn't strictly handling the toggling of the new button and it's more so creating the button so that it can be rendered on to the video player.  This makes it confusing to see this function get called in the initializing function.  With the above change, this function should be renamed to something like `createFullWindowButton` so that it better describes what the function is doing.  With this renamed, it can reference the new `toggleFullWindow` function or whatever it gets renamed to.